src/main/java/usi/si/seart/service/MetricLanguageService.java

```diff
@@ -0,0 +1,14 @@
+package usi.si.seart.service;
+
+import javax.validation.constraints.NotNull;
+import java.util.Optional;
+
+public interface MetricLanguageService {
+
+    /**
+     * Returns the ID of the provided metric language if it exists in the database.
+     * @param language_name the language name (e.g "C++")
+     * @return ID of the provided language if it exists in the database.
+     */
+    Optional<Long> getLanguageId(@NotNull String language_name);
```

Comment:
Java name conventions dictate the usage of camel case over snake case. I would stick to that for method parameters and fields.